test/e2e/page-objects/flows/snap-simple-keyring.flow.ts

```diff
@@ -1,18 +1,19 @@
 import { Driver } from '../../webdriver/driver';
 import SnapSimpleKeyringPage from '../pages/snap-simple-keyring-page';
-import { TEST_SNAPS_SIMPLE_KEYRING_WEBSITE_URL } from '../../constants';
 
 /**
  * Go to the Snap Simple Keyring page and install the snap.
  *
  * @param driver - The WebDriver instance used to interact with the browser.
  * @param isSyncFlow - Indicates whether to toggle on the use synchronous approval option on the snap. Defaults to true.
+ * @param port - The port number where the snap simple keyring site is running. Defaults to 8080.
  */
 export async function installSnapSimpleKeyring(
   driver: Driver,
   isSyncFlow: boolean = true,
+  port: number = 8080,
```

Comment:
ℹ️  we add this new param, to avoid port collisions in the case we need to use both the test dapp and the snap simple keyring page